tests/suites/0_stateless/02_0014_function_maths.sql

```diff
@@ -21,5 +21,7 @@ SELECT ln(1, 2); -- {ErrorCode 28}
 SELECT log10(NULL);
 SELECT log10(100);
 SELECT log2(2);
+SELECT mod(234, 10);
+SELECT mod(29, 9);
 
```

Comment:
It would be nice to add `select mod(34.5, 3)`

Reply:
Updated.